tests/conftest.py

```diff
@@ -16,14 +20,14 @@ def resources_dict():
     """
     resources_dict: Dict[str, Any] = {
         "data_category": models.DataCategory(
-            organization_fides_key=1,
+            organization_fides_key="1",
```

Comment:
https://docs.pydantic.dev/latest/migration/#validator-behavior-changes

An important change to behavior, integers are no longer automatically coerced to strings.  A lot of our FidesKeys in tests are an integer that were coerced to strings.  I believe these would be saved into the database as strings, so existing saved data is probably fine, but yaml files with integers as Fides keys would break.  This feels like something you would see in just test data though?

Reply:
Later, I ended up discovering "coerce_numbers_to_str", and added that to FidesModel to try to preserve some backwards compatibility here:

```
 model_config = ConfigDict(
        extra="ignore",
        from_attributes=True,
        coerce_numbers_to_str=True,  # Coercing for backwards compatibility with Pydantic v1
    )
```